### ISSUE TITLE
I cant open the zip file

### ISSUE THREAD

**Issue by BFDIFirey** [burnedpopcorn/PizzaTower-April2021-WebPort#1]
please help...

**Comment by burnedpopcorn** [comment]
are you on a chromebook or something?

**Comment by BFDIFirey** [comment]
yes did you forget?

**Comment by burnedpopcorn** [comment]
yeah i kind of did, i'll upload an uncompressed version

**Comment by BFDIFirey** [comment]
ok

**Comment by burnedpopcorn** [comment]
Its here: https://drive.google.com/drive/folders/1xZtEKDvALQtfYr1_B9jKlOrM9HtS5V-4?usp=sharing

**Comment by burnedpopcorn** [comment]
Press on the Folder Title, and press download

**Comment by BFDIFirey** [comment]
ok

**Comment by BFDIFirey** [comment]
do I download all?

**Comment by burnedpopcorn** [comment]
yeah, just delete index.html, because it doesn't load the game (it is always included when compiling , but never works)

**Comment by BFDIFirey** [comment]
ok

**Comment by BFDIFirey** [comment]
wait wtf I cant!?

**Comment by burnedpopcorn** [comment]
also you can delete runner.svg and runnerlocal.svg, and you can even replace them to display different images (just rename them with the same name)

**Comment by burnedpopcorn** [comment]
why cant you delete it? are you trying to delete it from the google drive link, because i'm talking about after you download it

**Comment by BFDIFirey** [comment]
so basically I tried deleting it but it says I cant

**Comment by burnedpopcorn** [comment]
if you're still in google drive, just download it, then in the Files app, then you can delete it

**Comment by BFDIFirey** [comment]
IM TRYING TO! (I'm not mad at you I'm mad at this stupid chromebook)

**Comment by burnedpopcorn** [comment]
shit that sucks, is it moving it to trash, or is it not deleting it at all

**Comment by BFDIFirey** [comment]
not deleting at all

**Comment by burnedpopcorn** [comment]
well, when i got my free account on x10hosting, it gave me a file manager for my server, so you could delete it there if the hosting site you use lets you

**Comment by burnedpopcorn** [comment]
you just need to find one tho, because of them not accepting new free accounts

**Comment by BFDIFirey** [comment]
wait let me see if deezboes can help

**Comment by BFDIFirey** [comment]
@deezboe

**Comment by burnedpopcorn** [comment]
holy shit thats a big file

**Comment by burnedpopcorn** [comment]
even x10hosting wouldn't cover that

**Comment by burnedpopcorn** [comment]
OH SHIT i found another service thats free, has unlimited bandwith, and has 5GB of storage

Its InfiniteFree ( https://www.infinityfree.com/ )

**Comment by BFDIFirey** [comment]
oh cool

**Comment by burnedpopcorn** [comment]
To compare, the game files are about 530mb, which x10hosting wouldn't even cover cuz their limit is 500mb

**Comment by burnedpopcorn** [comment]
also unlimited bandwidth is very important, because it can get very high (for example, for this project would use 530mb of bandwidth FOR EVERY PERSON THAT ENTERS THE SITE, so yeah it can very quickly reach 10s of gigabytes)

**Comment by BFDIFirey** [comment]
God damn!

**Comment by burnedpopcorn** [comment]
my 1.1.0 port has reached 2.01 Terabytes by itself, and its only been up for about 3 weeks

**Comment by burnedpopcorn** [comment]
honestly, i might switch to infinityfree just because its x10hosting but better and still free (500mb < 5GB simple maths)

**Comment by BFDIFirey** [comment]
yep I would too

**Comment by burnedpopcorn** [comment]
idk, i might use both if i have more projects i wanna host, x10 is good enough for now

**Comment by burnedpopcorn** [comment]
so, you got it hosted yet or you still working on it

**Comment by BFDIFirey** [comment]
still working I need to figure out how to make the game work

**Comment by burnedpopcorn** [comment]
what you should do is

1. get the files (easy for me for obvious reasons)
2. sign up for a web hosting service (you know i used x10hosting, but you're probably using infinitefree)
3. get a domain from the service
4. go to your site's file manager
5. put the files provided in the google drive folder into your public_html folder
6. you should be set

**Comment by burnedpopcorn** [comment]
because I don't use InfinityFree, i dont know the file system, so once you get an account, tell me what it says

**Comment by BFDIFirey** [comment]
I am so confused

**Comment by burnedpopcorn** [comment]
actually i might do it for you, cuz i already made an account, so why not

**Comment by BFDIFirey** [comment]
thank you I fucking tried to do it it says some stuff about domain checking idk wtf is going on

**Comment by burnedpopcorn** [comment]
it'll take a while tho

**Comment by BFDIFirey** [comment]
ok IDC it can take its time

**Comment by BFDIFirey** [comment]
how is it going so far?

**Comment by burnedpopcorn** [comment]
Alright, InfinityFree isn't working right now,  and I gotta do some stuff rn, so it'll take a while

**Comment by BFDIFirey** [comment]
ok

**Comment by deezboe** [comment]
> Alright, InfinityFree isn't working right now, and I gotta do some stuff rn, so it'll take a while

I had some trouble with inf free it technically has inf storage but you can't upload more then 100 mb as far as my experience went with it I used Amazon S3 but to verify you have to give them your phone number and card which sucks [I used a random visa gift card I had lying around]

**Comment by burnedpopcorn** [comment]
@deezboe Yeah, but its even worse than that. Upload Limit is 10mb for a single file, so there's no chance of me uploading the April build to InfinityFree (sorry @BFDIFirey, but I can't find a better provider for free). I was gonna make an alt account to get another free 500mb server from x10hosting, but I remembered that free sign-ups were shutdown, and either way the ports too big for it anyways, so yeah that sucks

**Comment by BFDIFirey** [comment]
x10hosting is back up

**Comment by burnedpopcorn** [comment]
Alright, I'll try a second account then

**Comment by burnedpopcorn** [comment]
Gonna take a while tho

**Comment by BFDIFirey** [comment]
ok

**Comment by burnedpopcorn** [comment]
Shit, alright so the game files are 540mb, and x10hosting's limit is 512mb, so we're kind of fucked

**Comment by burnedpopcorn** [comment]
My guess is the audio, and the possiblity of it not being compressed

**Comment by BFDIFirey** [comment]
first fmod didn't work on the noise update web port now something else damn this is annoying

**Comment by deezboe** [comment]
> My guess is the audio, and the possiblity of it not being compressed

yes the audio in Pizza Tower is not compressed and that's what makes it so big I could maybe show you how to compress the audio when I get home from boring ass school.

**Comment by burnedpopcorn** [comment]
Christ, the audio uncompressed is about 400mb by itself, Without the actual game files

**Comment by burnedpopcorn** [comment]
Since I compressed all the audio, it has now shrunk from 540mb to 100mb, so I'll be making a website for it

**Comment by burnedpopcorn** [comment]
Website is now Open

[http://dummydomain.x10.mx/pt-april-build/PizzaTower_GM2.html](http://dummydomain.x10.mx/pt-april-build/PizzaTower_GM2.html)

Enjoy

**Comment by BFDIFirey** [comment]
im going to probably stream this on twitch

**Comment by BFDIFirey** [comment]
gnome forest is broken

**Comment by burnedpopcorn** [comment]
Huh, I didn't even know it was in this build

**Comment by burnedpopcorn** [comment]
~~It might just be the level being in development, not sure~~

Nevermind, I see the problem (it crashes after you take the taxi, right?)

**Comment by BFDIFirey** [comment]
yes

**Comment by burnedpopcorn** [comment]
Yeah, not sure what I can do about that, as it could be the fact that this is a development build

**Comment by burnedpopcorn** [comment]
So it might not just be a gx.games issue

**Comment by BFDIFirey** [comment]
yeah makes sense knowing that the level still had bugs

**Comment by BFDIFirey** [comment]
also im going to make a cheesed up web port ill try to do it myself

**Comment by burnedpopcorn** [comment]
That's nice

**Comment by burnedpopcorn** [comment]
Tell me how that goes

**Comment by BFDIFirey** [comment]
well i need the launcher and stuff

**Comment by burnedpopcorn** [comment]
You can find leaked code of it on github if you search for it, and it depends on if they share the gamemaker version

**Comment by burnedpopcorn** [comment]
I've found a way to find the exact gamemaker version that was last used, so we don't really need to ask them either

**Comment by BFDIFirey** [comment]
im on chromebook

**Comment by burnedpopcorn** [comment]
Then how do you expect to make a port yourself if that's all you have?

**Comment by BFDIFirey** [comment]
idk

**Comment by burnedpopcorn** [comment]
Also i tried a cheesed up port, and that's not going so well. It compiles, but during startup it crashes, for either "Failing to start Rich Presence" or some other error when I try a different gamemaker version. My guess is the custom extensions included, and the fact that one of them has a critical online component which i'm guessing can't work because of some limitations with webassembly

**Comment by burnedpopcorn** [comment]
also the guy that made it apparently included an IP logger or something of that nature within that same project, so yeah ...

**Comment by BFDIFirey** [comment]
loypoll we hate you he did confirm he did

**Comment by burnedpopcorn** [comment]
Well it shouldn't matter here, because gx.games is in a sandbox, so I doubt it can send anything anyways

**Comment by burnedpopcorn** [comment]
Gonna update you on some stuff considering we haven't talked in this issue for a while

I've made a Cheesed Up Port, a Scoutdigo Port, and very recently a Sugary Spire Port

and I'm also gonna archive this repo, because I frankly don't care about this port
- but I will keep the website up
